common/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
pitty this is required but it's not too bad all things considered.

Reply:
Yeah, it could be worthwhile to have this auto-detect files and add them automatically, but I know some tools have trouble with dynamic requires (like webpack, I think).